### PR TITLE
create model

### PR DESCRIPTION
**Pull request by manjusakajin**
untitled

### CODE REVIEW

**Comment by zzvvmm** [comment]
Ok rùi

**Comment by PhanHongAnh** [comment]
okie

File: app/models/participation.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Participation < ApplicationRecord
+  enum accepted: { request: 0, join: 1 }
```
Comment (thanhhungchu95) [inline app/models/participation.rb]: xóa space thừa bên trong {} nhé, {request: 0, join: 1}

File: app/models/trip.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Trip < ApplicationRecord
+  belongs_to :place
+
+  has_many :participations, dependent: :destroy
+  has_many :members, through: :participations,
+    class_name: "User", foreign_key: "user_id"
```
Comment (DuongNT1512) [inline app/models/trip.rb]: User.name nhé

**Pull request merged**